### DISCUSSION TITLE
Allow players to see a slider speed before it starts

### DISCUSSION THREAD

**Discussion by PolskiKapsel** [ppy/osu#20382] [Ideas] [answered]
I would like to suggest a small feature which would allow players to see how fast would the next slider be. Nowadays if somebody's  playing the beatmap for the first time and the slider speed significantly changes, it leads into a break in most cases. All I would like to see is a small number below the slider number (x1,5 for example) that would allow players to adapt into the new slider speed. It surely would be confusing at first, however with enough experience players would learn how fast it is. 

![Slider Speed](https://user-images.githubusercontent.com/102519385/191356469-e920ab24-294c-426a-b63c-26da1cb7f905.png)

Ps. I also would like to apologize if I made any spelling mistakes. English is not my primary language, however I really wanted to share this idea with others

**Comment by Feodor0090** [comment, marked as answer]
IMO an idea from #13729 will work better than random floating numbers which you are supposed to read and understand during less than a second. Or maybe some animations on slider body, which speed depends on slider's speed in that part...

**Reply by PolskiKapsel** [reply]
Honestly? Yeah, I think so too. I just couldn't come up with a better idea for some kind of an Indicator that would help players match their cursor speed with slider. I must have missed this idea while checking up if something similar was already suggested.
